src/Compiler/FSharp.Compiler.Service.fsproj

```diff
@@ -25,12 +25,19 @@
     <FsYaccOutputFolder>$(IntermediateOutputPath)$(TargetFramework)\</FsYaccOutputFolder>
     <FsLexOutputFolder>$(IntermediateOutputPath)$(TargetFramework)\</FsLexOutputFolder>
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
-    <Configurations>Debug;Release;ReleaseCompressed</Configurations>
+    <Configurations>Debug;Release</Configurations>
   </PropertyGroup>
 
   <!--  The FSharp.Compiler.Service dll provides a referencable public interface for tool builders -->
   <PropertyGroup Condition="'$(Configuration)' != 'Proto'">
-    <CompressMetadata Condition="'$(CompressAllMetadata)' != 'true'">false</CompressMetadata>
+    <NoOptimizationData>false</NoOptimizationData>
+    <NoInterfaceData>false</NoInterfaceData>
+    <CompressMetadata>false</CompressMetadata>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' != 'Proto' and '$(BUILDING_USING_DOTNET)' != 'true'">
+    <OtherFlags Condition="'$(BUILDING_USING_DOTNET)' != 'true' and '$(TestingLegacyInternalSignature)' != 'true'">$(OtherFlags) --realsig-</OtherFlags>
```

Comment:
We will start using the realsig build tag, once the rtm SDK and VS contains a buildtask that recognizes it.  Otherwise it's just too confusing to keep straight.
However, I can certainly remove the redundent '$(BUILDING_USING_DOTNET)' != 'true' tests.